### ISSUE TITLE
Use #DeleteGithub to Label Your Migrated Projects

### ISSUE THREAD

**Issue by biergaizi** [upend/IF_MS_BUYS_GITHUB_IMMA_OUT#29]
As many people has decided to migrate their projects to other platforms, why not to label the abandoned GitHub project as "#DeleteGithub"?

If we use #DeleteGithub tag to label our migrated project, and encourages other to do the same, we can easily see how many project has chosen to migrate by searching this tag. Like this.

We can make the transition as smooth as possible. You don't have to actually delete the repo and you don't have to break things for others, you can just

    git branch deleted
    git checkout deleted
    git rm -rf *
    echo '===#DeleteGithub===' > README.md
    echo '**Deleted!** This project is moved to [notabug.org](https://notabug.org/example/example), welcome to join the free world!' > README.md
    git add README.md
    git commit -a
    git push --set-upstream origin deleted

Then, click "Settings" -> "Branches" -> make "deleted" as Default branch. Finally add the tag "#deletegithub" to your project.

**Comment by lanodan** [comment]
> Also, please if you do move your code away from Github:
Be sure that your package maintainers are aware of this change, for example by filling an issue.

Haelwenn /ɛlwən/ — https://social.hacktivis.me/objects/a38b05b2-dc86-4f8d-8664-41d3f2669f7f

This technique of yours doesn’t make a notification, and you’ll likely delete your account soon. Potentially breaking the release checks of package maintainers and without having any kind of information after that if you don’t have a proper homepage or channel.

**Comment by biergaizi** [comment]
Good point. Silently breaking automatic release checks relied by package maintainers is a serious issue. I guess for a influential project, converting the GitHub repo to a pure mirror may be a better solution.

**Comment by 0100001001000010** [comment]
I put #DeleteGithub on my profile, but I am not deleting my account in case I find a project I want to contribute to that is still hosted on GitHub.

**Comment by biergaizi** [comment]
@lanodan Perhaps intentionally creating a non-working version tag may help solving this problem. It's not the nicest thing to do, but it does trigger a new version alert, and likely to cause the build process to fail, in this process the maintainers are notified.

**Comment by lanodan** [comment]
@biergaizi 
Makes me think that “destroying” the git repo probably isn’t a good thing either for things based on git (archlinux ``${pkgname}-git`` and gentoo’s ``${pkgname}-…9999`` packages).

And yeah, creating a fake release (and so tag) is probably a bad idea for stuff like that too. (I have ones using the tag nearest the HEAD in the tree because there is too much releases)

**Comment by DavidGriffith** [comment]
There are a goodly number of repos using this label that are pointing to new repos on Gitlab that aren't marked public.  You need to explicitly do that.

**Comment by vassudanagunta** [comment]
Hey guys. I love the work you've been doing. Not everyone loves mine. Please read Issue 114.

**Comment by vassudanagunta** [comment]
Hello,

The GitHub Evacuation Project has moved to GitLab (not an endorsement or even a final home). Your enthusiasm and contribution is still needed. Please check out [the new project home](https://gitlab.com/upend/github), and read [the project wiki](https://gitlab.com/upend/github/evac-guide/wikis) for info to get restarted.

Thank you!

vas

**Comment by vassudanagunta** [comment]
@biergaizi @lanodan @0100001001000010 @DavidGriffith We need to incorporate your ideas into the [Evacuation Guide](https://gitlab.com/upend/github/evac-guide), probably under [Covering all your bases](https://gitlab.com/upend/github/evac-guide/blob/master/covering-all-your-bases.md). Can you guys start that?

Also, would you guys also change your profile photo and add "github evacuee" to your profile and repos as I wrote under [Help move the movement](https://github.com/upend/IF_MS_BUYS_GITHUB_IMMA_OUT#help-move-a-movement)? I believe a consistency of imagery and message has the best chance of catching fire, like the original repo did before we got buried from the trending list by Microsoft GitHub.

**Comment by bruceblore** [comment]
I won't add github evacuee to all of my repos because they are already archived and undoing that will be a lot of work. I just added it to my profile and a repo telling people where I am going. I will delete my repos from Github at some point. People are more likely to notice an error on push or pull than they are to notice a new tag. I think I will put an obnoxiously huge and flashing "#DeleteGithub" on my website though.

**Comment by vassudanagunta** [comment]
@0100001001000010 Thanks! You show up in [this search result](https://github.com/search?q=github+evacuee&type=Users). Just started this a day ago. Hoping it starts growing geometrically.

**Comment by vassudanagunta** [comment]
@0100001001000010 could you put the steps you took to evacuate in the Guide?

**Comment by vassudanagunta** [comment]
@0100001001000010 thanks for the additions to the Guide. I merged them (after fixup).
